integrations/lambda/src/lambda_sample/lambda_helper.py

```diff
@@ -0,0 +1,296 @@
+from boto3 import session
+from botocore.auth import SigV4Auth
+from botocore.awsrequest import AWSRequest
+from http import HTTPStatus
+from requests.exceptions import HTTPError
+import json
+import os
+import requests
+import time
+import zipfile
+
+MAX_WAIT_SECONDS = 900 # 15 minutes
+
+def create_lambda(session: session, lambda_name: str, database_name: str, table_name: str, role_name: str,
+                  mem_store_retention_period_in_hours=12, mag_store_retention_period_in_days=3653, batch_size=100) -> str:
```

Comment:
either this default is a typo or we care a lot about retaining this data 😂 
```suggestion
                  mem_store_retention_period_in_hours=12, mag_store_retention_period_in_days=365, batch_size=100) -> str:
```

Reply:
When you create a table in Timestream for LiveAnalytics, this is the default value for the mag store retention period (10 years) rounded up.